.changeset/curly-tools-care.md

```diff
@@ -0,0 +1,5 @@
+---
+'explorer': minor
+---
+
+Replaced Siacentral with Explored on the Block route.
```

Comment:
Two notes on the changeset:
- The two names are styled "Sia Central" (with a space) and explored (lowercased). 
- For the app changesets, since these are used as a sort of change log of new features for non-coder users, try to make the changesets present tense and from the perspective of the user / less technical details when/if possible. eg: `The explorer now uses explored for block data.`.